src/main.js

```diff
@@ -20,17 +28,100 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
-function displayNewCover() {
+function displayNewCover() {  
   bookCover.src = covers[getRandomIndex(covers)];
   bookTitle.innerText = titles[getRandomIndex(titles)];
   firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
   secondDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
+  currentCover = new Cover(bookCover.src, bookTitle.innerText, firstDescriptor.innerText, secondDescriptor.innerText);
+}
+
+//Iteration 2: 
+
+
+//Form-View Functionality:
+
+var makeNewButton = document.querySelector('.make-new-button');
+makeNewButton.addEventListener('click', toggleMakeOwn);
+
+
+function toggleMakeOwn() {
+  document.querySelector('.random-cover-button').classList.add('hidden');
+  document.querySelector('.save-cover-button').classList.add('hidden');
+  document.querySelector('.home-button').classList.remove('hidden');
+  document.querySelector('.form-view').classList.remove('hidden');
+  document.querySelector('.home-view').classList.add('hidden');
+  document.querySelector('.saved-view').classList.add('hidden');
+}
+
+//Save-Cover Functionality:
+
```

Comment:
We'll have to go back and remove these comments when we do our final refactor on Sunday.